sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -515,18 +515,16 @@ export class JsonRpcProvider extends Provider {
     options?: SuiObjectDataOptions,
```

Comment:
I don't think we have any test cases covering `getObjectBatch`, do you mind adding one in objects.test.ts?